scripts/install.js

```diff
@@ -21,32 +21,37 @@ require('../lib/extensions');
  */
 
 function download(url, dest, cb) {
-  var returnError = function(err) {
-    cb(typeof err.message === 'string' ? err.message : err);
+  var reportError = function(err) {
+    cb(['Cannot download "', url, '": ',
+      typeof err.message === 'string' ? err.message : err].join(''));
```

Comment:
Would this not cause an error if `err.message` was undefined? i.e. if `err` was a string